osxkeychain.go

```diff
@@ -34,6 +34,7 @@ type GenericPasswordAttributes struct {
 	AccountName         string
 	Password            []byte
 	TrustedApplications []string
+	Keychain            []string
```

Comment:
Yeah, I ran into this issue when trying to add more attributes. I'll create another pull request with this in it.